clients/src/main/java/org/apache/kafka/clients/consumer/ConsumerRebalanceListener.java

```diff
@@ -122,4 +122,30 @@
      * @throws org.apache.kafka.common.errors.InterruptException If raised from a nested call to {@link KafkaConsumer}
      */
     void onPartitionsAssigned(Collection<TopicPartition> partitions);
+
+    /**
+     * A callback method the user can implement to provide handling of cleaning up resources for partitions that have already
```

Comment:
```suggestion
     * A callback method you can implement to provide handling of cleaning up resources for partitions that have already
```

This javadoc is talking to "the user" ;)